zou/app/blueprints/previews/resources.py

```diff
@@ -338,8 +338,6 @@ def save_movie_preview(
             preview_files_service.prepare_and_store_movie(
                 preview_file_id, uploaded_movie_path, normalize=normalize
             )
-            preview_file = files_service.get_preview_file(preview_file_id)
-            tasks_service.update_preview_file_info(preview_file)
```

Comment:
why do you remove the metedata from the preview file?

Reply:
Because it's already done in the function before : https://github.com/cgwire/zou/blob/master/zou/app/services/preview_files_service.py#L305

Reply:
So it was done twice